content/en/developers/dogstatsd/datagram_shell.md

```diff
@@ -74,6 +74,24 @@ The container ID is prefixed by `c:`, for example:
 
 Read more about container tags in the [Kubernetes][4] and [Docker][5] tagging documentation.
 
+### DogStatsD protocol v1.3
+
+Starting with the Agent `>=v6.40.0` or `>=v7.40.0`, a new optional Unix timestamp field is supported.
+
+When this field is provided, the Datadog Agent doesn't do any processing with the metrics (i.e. no aggregation) except from enriching the metrics with tags. This can be useful is you already aggregate your metrics in your application and you want to send them to Datadog without extra processing.
```

Comment:
```suggestion
When this field is provided, the Datadog Agent doesn't do any processing with the metrics (no aggregation) except from enriching the metrics with tags. This can be useful if you already aggregate your metrics in your application, and you want to send them to Datadog without extra processing.
```